README.md

```diff
@@ -45,15 +59,36 @@ To test the second store (alone or in conjunction) change the metric path to `te
 Record structure
 ----------------
 
-The only supported protocol is [line](https://graphite.readthedocs.io/en/latest/feeding-carbon.html#the-plaintext-protocol). The records should follow the structure:
+The only supported protocol is [line](https://graphite.readthedocs.io/en/latest/feeding-carbon.html#the-plaintext-protocol). The records have the structure:
 ```
 path.path.path value datetime
 ```
 
 Config
 ------
 
-Please refer to the sample configs in the _config_ folder for examples and documentation.
+Please refer to the [sample config](config/config.toml) for examples and documentation.
+
+Routing
+-------
+
+Is defined in the [rules config](config/rules.toml) that is in turn referred to in the [main config](config/config.toml). This is how it works:
+- routing rules are applied to each incoming record in order;
+- `continue` variable defines whether to continue matching next rules. It is `true` by default;
+- when regex in a rule matches, the record is sent to the clusters in the `clusters` list;
+- multiple rules can be matched to each record;
+- each record can be sent to a single cluster at most once. If two rules send it to same cluster, only one instance will be sent;
+- cluster names must be from the set defined in the [clusters config](clonfig/clusters.toml);
+
+#### Rewrites
+
+Optionally, it is possible to apply the [rewrite rules](config/rewrite.toml). This is how they work:
+
+- all rules are applied to each record;
+- rewrites are applied before the routing;
+- rule matches if `from` matches;
+    - then metric path is rewriten to `to`;
+    - if `copy` is `true` the original metric is retained in addition to the modified one.
```

Comment:
Please clarify that rewritten results are passed directly to the routing 'rules' bypassing rewrites.
Unclear: can multiple 'from' match create multiple 'to' metrics? Does copy = 'true'/'false' affect the answer?

Reply:
This is why I think we should not have re-writes. They are not usable and opaque. They also create a lot of "magic": I sent A, but got B, but why? I hope we will remove them in the future.

I will add more explanations.